CONTRIBUTING.md

```diff
@@ -97,3 +97,4 @@ exact = ["~/code/git-bug/.envrc"]
 [install/nix]: https://nix.dev/install-nix
 [install/direnv]: https://github.com/direnv/direnv/blob/master/docs/installation.md
 [install/direnv/shell]: https://github.com/direnv/direnv/blob/master/docs/hook.md
+[install/install-direnv-without-nix]: #user-content-install-direnv-without-nix
```

Comment:
```suggestion
[install/install-direnv-without-nix]: #install-direnv-without-nix
```

i'm not quite sure where the `user-content` bit comes from (github's preview ui, maybe?); the above suggestion should work.